### PR TITLE
pruntime: Allow to reset operator in GetRuntimeInfo

### PR DESCRIPTION
**Pull request by kvinwang**
@jasl Can you test this PR? I haven't test it yet.

To test:
- [ ] Reseting operator should just work. (To test it, just restart pherry with a new operator).
- [ ] Should be able to load previous pruntime saved checkpoint.
- [ ] Old pherry/prb should be able to talk to new pruntime.
- [ ] New pherry/prb should be able to talk to old pruntime.

To do:
- [ ] Back port the pruntime maintaince branch.
- [ ] Bump versions.

### CODE REVIEW

**Review by h4x3rotab — APPROVED** [review]
LGTM

**Comment by jasl** [comment]
I shall merge first, previously I found master branch PRuntime can't sync Khala chain, let me try again.

Also, please backport this, so we can test on PC3

**Comment by h4x3rotab** [comment]
@jasl Could you help with the back port? Thanks

**Comment by jasl** [comment]
> @jasl Could you help with the back port? Thanks

let's @kvinwang do this
